DNN Platform/Library/Common/Utilities/UrlUtils.cs

```diff
@@ -219,8 +219,13 @@ public static void OpenNewWindow(Page page, Type type, string url)
 
         public static string PopUpUrl(string url, Control control, PortalSettings portalSettings, bool onClickEvent, bool responseRedirect)
         {
-            return PopUpUrl(url, control, portalSettings, onClickEvent, responseRedirect, 550, 950);
```

Comment:
This could be a breaking change for other modules using this method

Reply:
Should not be. There is another method overloading. 
` public static string PopUpUrl(string url, Control control, PortalSettings portalSettings, bool onClickEvent, bool responseRedirect, String closingUrl)
        {
            return PopUpUrl(url, control, portalSettings, onClickEvent, responseRedirect, 550, 950, true, closingUrl);
        }`
the exact same method with closingUrl = string.empty. The default behavior for closingURL is string.empty.

Reply:
You are correct, sorry about that.